x/concentrated-liquidity/internal/math/tick_test.go

```diff
@@ -13,22 +13,22 @@ func (suite *ConcentratedMathTestSuite) TestTickToSqrtPrice() {
 	}{
 		"positive tick index 1": {
 			tickIndex:         sdk.NewInt(85176),
-			sqrtPriceExpected: "70.710004849206351867", // 70.710004849206120647
+			sqrtPriceExpected: "70.710004849206120646",
```

Comment:
The values seem slightly off judging by the old comment.

I think we should specify `sqrtPriceExpected` as given by Wolfram and define an "additive tolerance" for each test case.

Reply:
old comment was actually off (rounded instead of truncated like we want).